java/core/src/main/java/co/worklytics/psoxy/utils/RandomNumberGenerator.java

```diff
@@ -0,0 +1,7 @@
+package co.worklytics.psoxy.utils;
+
+public interface RandomNumberGenerator {
```

Comment:
interface used to hide `Random`, which can't be mocked with mockito in jdk17+ (until Mockito 5)